### PR TITLE
reference: Refactor docs about structure

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

**Comment by gaocegege** [comment]
http://stat.ethz.ch/R-manual/R-patched/library/base/html/00Index.html

This is the doc about R, we need to migrate some docs from there

File: examples/reference/def/.property.yml [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 description: "
-    The keyword <b>def</b> introduces a function definition. It is followed by a function name, and a paranthesized list of parameters.  
+The names in an argument list can be back-quoted non-standard names.
+
+If value is missing, NULL is returned. If it is a single expression, the value of the evaluated expression is returned. (The expression is evaluated as soon as return is called, in the evaluation frame of the function and before any on.exit expression is evaluated.)
+
+If the end of a function is reached without calling return, the value of the last evaluated expression is returned.
 "
-syntax: "	def functionName():\n	    statements"
+syntax: "function( arglist ) expr \nreturn(value)"
```
Comment (jeremydouglass) [inline examples/reference/def/.property.yml]: Should examples/reference/def be renamed examples/reference/function ?

File: examples/reference/importLibrary/.property.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-category: Library
+category: Structure
 subcategory: 
 description: "
-Import Processing libraries.
+	The keyword importLibrary is used to load a library into a Processing sketch. A library is one or more classes that are grouped together to extend the capabilities of Processing.
```
Comment (jeremydouglass) [inline examples/reference/importLibrary/.property.yml]: Great!

File: examples/reference/def/.property.yml (hunk 1 above)
Comment (gaocegege) [inline examples/reference/def/.property.yml]: Yeah, it is my mistake!

**Pull request merged**